libs/subliminal_patch/providers/whisperai.py

```diff
@@ -270,9 +295,11 @@ def query(self, language, video):
 
         if sub.task == "translate":
             if language.alpha3 != "eng":
-                logger.info(f"Translation only possible from {language} to English")
+                logger.debug(f"Translation only possible from {language} to English")
                 return None
-
+            
+        # tell the user what we are about to do
+        sub.release_info = f"{sub.task} {language_from_alpha3(sub.audio_language)} audio -> {language_from_alpha3(language.alpha3)} SRT"
```

Comment:
Any reason why we can't use subzero.Language for this? I'm concerned about it breaking on some weird inconsistency.

Reply:
No reason, except that I didn’t know it existed… 😀 
(I’m still learning about all of the libraries)
That was even easier than pycountry.
Fixed.